arlas-core/src/main/java/io/arlas/server/utils/CheckParams.java

```diff
@@ -86,6 +86,16 @@ else if (request != null) {
         }
     }
 
+    public static void checkCountDistinctRequest(CountDistinct request) throws ArlasException {
+        if (request == null) {
+            throw new BadRequestException("CountDistinct request should not be null");
+        } else {
```

Comment:
You don't need the else, of you could write else if to avoid a condition level

Reply:
Done :+1: